### PR TITLE
default to universe version-2.x during install

### PR DESCRIPTION
**Pull request by tamarrow**

### CODE REVIEW

**Comment by jsancio** [comment]
Let's update the Windows installer too. Thanks!

**Comment by jsancio** [comment]
Fixes #480

File: win_bin/install/install-dcos-cli.ps1 [hunk 1]
```diff
@@ -104,7 +104,7 @@ dcos config set core.reporting true
 dcos config set core.dcos_url $dcos_url
 dcos config set core.timeout 5
 dcos config set package.cache $env:temp\dcos\package-cache
-dcos config set package.sources '[\"https://github.com/mesosphere/universe/archive/version-1.x.zip\"]'
+dcos config set package.sources '[\"https://github.com/mesosphere/universe/archive/version-2.x.zip\"]'
```
Comment (tamarrow) [inline win_bin/install/install-dcos-cli.ps1]: @jsancio the windows installer is updated here!

**Comment by jsancio** [comment]
LGTM! :shipit:

File: win_bin/install/install-dcos-cli.ps1 (hunk 1 above)
Comment (jsancio) [inline win_bin/install/install-dcos-cli.ps1]: :+1: I guess I need more coffee.

**Comment by tamarrow** [comment]
@jsancio updated to use `https://universe.mesosphere.com/repo`

**Comment by jsancio** [comment]
Great. Thanks you. LGTM. :shipit:

**Pull request merged**